File: programacao/c/respostas/operacoes/36.c
#include <stdio.h>

int main() {
    float valor_tipo1, valor_tipo2;
    int qtd_tipo1, qtd_tipo2;
    float total;

    printf("Digite o valor do reembolso para cada garrafa do tipo 1: ");
    scanf("%f", &valor_tipo1);

    printf("Digite o valor do reembolso para cada garrafa do tipo 2: ");
    scanf("%f", &valor_tipo2);

    printf("Digite a quantidade de garrafas do tipo 1: ");
    scanf("%d", &qtd_tipo1);

    printf("Digite a quantidade de garrafas do tipo 2: ");
    scanf("%d", &qtd_tipo2);

    total = valor_tipo1 * qtd_tipo1 + valor_tipo2 * qtd_tipo2;

    printf("O total do reembolso e R$ %.2f\n", total);

    return 0;
}

File: programacao/python/respostas/operacoes/36.py
valor_tipo1 = float(input("Valor do reembolso para cada garrafa do tipo 1: "))
valor_tipo2 = float(input("Valor do reembolso para cada garrafa do tipo 2: "))
qtd_tipo1 = int(input("Quantidade de garrafas do tipo 1: "))
qtd_tipo2 = int(input("Quantidade de garrafas do tipo 2: "))

total = valor_tipo1 * qtd_tipo1 + valor_tipo2 * qtd_tipo2
print(f"O total do reembolso e R$ {total:.2f}")
